photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -437,6 +451,71 @@ private Optional<EstimatedRobotPose> multiTagOnCoprocStrategy(
         }
     }
 
+    private Optional<EstimatedRobotPose> imuTagOnRioStrategy(
+            PhotonPipelineResult result, Optional<Matrix<N3, N3>> cameraMatrixOpt) {
+        if (result.hasTargets() && cameraMatrixOpt.isPresent()) {
+            PhotonTrackedTarget target = result.getBestTarget();
+
+            List<TargetCorner> targetCorners = target.getDetectedCorners();
+            double sumX = 0.0;
+            double sumY = 0.0;
+            for (TargetCorner t : targetCorners) {
+                sumX += t.x;
+                sumY += t.y;
+            }
+
+            Point tagCenter = new Point(sumX / 4, sumY / 4);
+
+            Rotation3d tagAngle = PhotonUtils.correctPixelRot(tagCenter, cameraMatrixOpt.get());
+
+            int tagID = target.getFiducialId();
+            Optional<Pose3d> tagPose = fieldTags.getTagPose(tagID);
+
+            if (tagPose.isEmpty()) {
+                reportFiducialPoseError(tagID);
+                return Optional.empty();
+            }
+
+            // calculate distance to the tag
+            double hypot =
+                    -PhotonUtils.calculateDistanceToTargetMeters(
+                            robotToCamera.getZ(), tagPose.get().getZ(), robotToCamera.getY(), tagAngle.getY());
```

Comment:
Shouldn't `robotToCamera.getY()` be `robotToCamera.getRotation().getY()`? Or is this intentional?

Reply:
I think you're right, that looks wrong. I think I may have fixed that locally during testing awhile back, but I'll make sure it gets fixed when I rewrite to use the PhotonUtils function.